.github/workflows/ci.yaml

```diff
@@ -30,6 +28,29 @@ jobs:
         node-version: ${{ matrix.node-version }}
     - uses: bahmutov/npm-install@v1
     - run: yarn format:check
+  eslint-check:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
```

Comment:
I think you can lift the `strategy` up to a higher level in the YAML to avoid repeating this matrix definition for each job:

https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs

Reply:
Wait, or maybe not. I may have misread this...

Reply:
Oh sorry not very familiar with these. But which level can we push this strategy to?

Reply:
Oh okay. Saw ur comment just in time.

Reply:
Nevermind, I think it can only be done easily here for steps within a job as you've done, not for all the jobs within a workflow. There is probably some way to do it using a matrix with reusable workflows or something but I'm not sure and it may not be worth the effort for such a relatively small and simple workflow file.